client/state/happychat/selectors/can-user-send-messages.js

```diff
@@ -19,13 +26,4 @@ import isHappychatClientConnected from 'calypso/state/happychat/selectors/is-hap
  */
 export default ( state ) =>
```

Comment:
When doing refactors like this, I'm also trying to migrate away from anonymous functions and converting them to something like `export default function canUserSendMessages`. That makes the selector easier to find with grep, and maybe also avoid seeing `(anonymous)` stack frames in devtools.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/60043/commits/6783ef02450723e66e5f06371113f1141592ed7c